lib/rubocop/cop/lint/raise_exception.rb

```diff
@@ -5,33 +5,64 @@ module Cop
     module Lint
       # This cop checks for `raise` or `fail` statements which are
       # raising `Exception` class.
+      # You can specify a module name that will be an implicit namespace
```

Comment:
Probably the explanation should be extended to mention the cop normally ignores namespaced "Exception", but might cause false positives if some module has its own `Exception` class. This would make it easier for people to understand the need for the extra config. That being said - I think it's a very bad idea to name base classes "Exception" and I'd rather name them "(BaseError)" or something along those lines. Hiding built-in constants is a recipe for trouble IMO.

Reply:
Thank you for reviewing. I extended the explanation.